javascript-view-swapping/javascript-view-swapping-quiz-notes.md

```diff
@@ -0,0 +1,55 @@
+# javascript-view-swapping-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What is the `event.target`?
+the element that the event listener is triggered from;
```

Comment:
Not quite! `event.currentTarget` refers to the element that has the event listener attached to it, but `event.target` refers to the actual element that was interacted with to trigger the event